os_migrate/plugins/module_utils/workload_common.py

```diff
@@ -148,6 +116,39 @@ def __init__(self, openstack_connection, conversion_host_id, ssh_key_path,
         # Ports chosen for NBD export
         self.claimed_ports = []
 
+    def check_process(self, pid):
+        """Check whether pid exists in the current process table."""
+        if pid is None:
```

Comment:
If we don't do input validation before calling the method (below on line 142), maybe we should handle getting an empty string here too, just in case the .lock is from an invocation of old OSM which didn't yet write the PID there? Perhaps

```
if not pid:
    return False
```